frontend/packages/ceph-storage-plugin/src/components/independent-mode/install.tsx

```diff
@@ -52,10 +53,19 @@ const CreateExternalCluster = withHandlePromise((props: CreateExternalClusterPro
   const [clusterServiceVersion, setClusterServiceVersion] = React.useState(null);
   const [fileData, setFileData] = React.useState('');
   const [dataError, setDataError] = React.useState('');
+  const [ipFamily, setIPFamily] = React.useState(IP_FAMILY.IPV4);
   const dispatch = useDispatch();
 
   const plainKeys = _.concat(configMaps, storageClasses);
 
+  React.useEffect(() => {
+    // eslint-disable-next-line promise/catch-or-return
+    k8sList(PodModel).then((pods) => {
+      const address = pods[0].status.podIP;
+      setIPFamily(getIPFamily(address));
+    });
```

Comment:
instead of disabling the eslint, you can do `setIPFamily(IP_FAMILY.IPV4` in catch block. We should disable eslint only if required.

Reply:
but it's already IPv4 by default. I don't have good enough reason to add a catch block.